Codellama/README.md

```diff
@@ -0,0 +1,15 @@
+# CodeLlama 7B Local Test 
+
+1. Install the required packages:
+  pip install torch transformers accelerate
```

Comment:
I think it's customary to use some sort of dependencies file in Python, could you look into setting that up?